src/mcp_atlassian/server.py

```diff
@@ -877,6 +877,41 @@ async def list_tools() -> list[Tool]:
                         },
                     },
                 ),
+                Tool(
+                    name="jira_create_sprint",
+                    description="Create Jira sprint for a board",
+                    inputSchema={
+                        "type": "object",
+                        "properties": {
+                            "board_id": {
+                                "type": "string",
+                                "description": "The id of board (e.g., '1000')",
+                            },
+                            "sprint_name": {
+                                "type": "string",
+                                "description": "Name of the sprint (e.g., 'Sprint 1')",
+                            },
+                            "start_date": {
+                                "type": "string",
+                                "description": "Start time for sprint (ISO 8601 format)",
+                            },
+                            "end_date": {
+                                "type": "string",
+                                "description": "End time for sprint (ISO 8601 format)",
+                            },
+                            "goal": {
+                                "type": "string",
+                                "description": "Goal of the sprint",
+                            },
+                        },
+                        "required": [
+                            "board_id ",
```

Comment:
There's an extra space in "board_id " in the required fields list. This could cause validation issues.

```suggestion
                            "board_id",
```

Reply:
good catch!